File: sdks/csharp/sdk/src/PogodocApi/Documents/Types/GetJobStatusResponseType.cs
using System.Runtime.Serialization;
using System.Text.Json.Serialization;
using PogodocApi.Core;

namespace PogodocApi;

[JsonConverter(typeof(EnumSerializer<GetJobStatusResponseType>))]
public enum GetJobStatusResponseType
{
    [EnumMember(Value = "docx")]
    Docx,

    [EnumMember(Value = "xlsx")]
    Xlsx,

    [EnumMember(Value = "pptx")]
    Pptx,

    [EnumMember(Value = "ejs")]
    Ejs,

    [EnumMember(Value = "html")]
    Html,

    [EnumMember(Value = "latex")]
    Latex,

    [EnumMember(Value = "react")]
    React,
}

File: sdks/ruby/src/sdk/lib/pogodoc_api_client/documents/types/get_job_status_response_type.rb
# frozen_string_literal: true

module PogodocApiClient
  class Documents
    # Type of template to be rendered
    class GetJobStatusResponseType
      DOCX = "docx"
      XLSX = "xlsx"
      PPTX = "pptx"
      EJS = "ejs"
      HTML = "html"
      LATEX = "latex"
      REACT = "react"
    end
  end
end
